base/commands/map/map_destroy.go

```diff
@@ -0,0 +1,65 @@
+package _map
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/prompt"
+)
+
+const (
+	mapDestroyFlagAutoYes = "yes"
+)
+
+type MapDestroyCommand struct{}
+
+func (mc *MapDestroyCommand) Init(cc plug.InitContext) error {
+	help := "Destroy a Map"
+	cc.SetCommandHelp(help, help)
+	cc.AddBoolFlag(mapDestroyFlagAutoYes, "", false, false, "skip interactive approval")
+	cc.SetCommandUsage("destroy")
+	return nil
+}
+
+func (mc *MapDestroyCommand) Exec(ctx context.Context, ec plug.ExecContext) error {
+	mv, err := ec.Props().GetBlocking(mapPropertyName)
```

Comment:
How about `m, err := ec.Props().GetBlocking(mapPropertyName).(*hazelcast.Map)` to save a line?